src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ObjectCreationCompletionProviderTests.cs

```diff
@@ -437,5 +437,53 @@ void M1()
 ";
             await VerifyItemExistsAsync(markup, "object");
         }
+
+        [WorkItem(14084, "https://github.com/dotnet/roslyn/issues/14084")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task InMethodCallBeforeAssignment1()
+        {
+            var markup =
+@"namespace ConsoleApplication1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            object o;
+            string s;
+
+            Test(new $$
+            o = s;
+        }
+        static void Test(TimeSpan t, TimeSpan t2) { }
+    }
+}
+";
+            await VerifyItemExistsAsync(markup, "TimeSpan");
```

Comment:
These tests only test a single provider at a time, so yes, TimeSpan only appears because of preselection. Of course in this case, it's an errortypesymbol.